common/djangoapps/mailchimp_pipeline/tasks.py

```diff
@@ -12,20 +12,21 @@
 
 
 @task()
-def update_org_details_at_mailchimp(org_label, org_type, work_area, list_id):
+def update_org_details_at_mailchimp(org_label, org_type, work_area, org_id,  list_id):
     log.info("Task to send organization details to MailChimp")
     log.info(org_label)
 
-    extended_profiles = UserExtendedProfile.objects.filter(organization__label__iexact=org_label.lower()).values("user__email")
+    extended_profiles = UserExtendedProfile.objects.filter(organization_id=org_id).values("user__email")
 
-    for extended_profile in extended_profiles:
-        user_json = {
-            "merge_fields": {
-                "ORG": org_label,
-                "ORGTYPE": org_type,
-                "WORKAREA": work_area
-            }
+    user_json = {
+        "merge_fields": {
+            "ORG": org_label,
+            "ORGTYPE": org_type,
+            "WORKAREA": work_area
```

Comment:
please use single quotes

Reply:
This will cause inconsistency with the rest of the file/module, so I avoided doing that and still strongly advice against it, since that is not in the scope of the ticket, too.